src/main/scala/vectorpipe/osm/package.scala

```diff
@@ -38,7 +40,21 @@ package object osm {
     * If you want to read a file from S3, you must call [[vectorpipe.useS3]] first
     * to properly configure Hadoop to read your S3 credentials.
     */
-  def fromORC(path: String)(implicit ss: SparkSession): Either[String, (RDD[Node], RDD[Way], RDD[Relation])] = {
+  def fromORC(
+    path: String
+  )(implicit ss: SparkSession): Either[String, (RDD[(Long, Node)], RDD[(Long, Way)], RDD[(Long, Relation)])] = {
```

Comment:
All three of these functions now return a paired `Long` as well to aid partitioning. Borrowed from @lossyrob .